Polyhedron/demo/Polyhedron/Scene_polyhedron_item_k_ring_selection.h

```diff
@@ -146,6 +189,33 @@ public Q_SLOTS:
     updateIsTreated();
   }
 
+  // slots are called by signals of surface_mesh_item
+  void sm_vertex_has_been_selected(void* v)
+  {
+    std::size_t h = reinterpret_cast<std::size_t>(v);
+    is_active=true;
+    if(active_handle_type == Active_handle::VERTEX || active_handle_type == Active_handle::PATH)
+      process_selection( static_cast<sm_vertex_descriptor>(h) );
```

Comment:
Sorry if this is part of PR #1983 but I compiled this branch. This line causes a conversion warning from `std::size_t` to `CGAL::SM_Index<CGAL::SM_Vertex_index>::size_type` here